Dockerfile

```diff
@@ -40,38 +41,42 @@ WORKDIR /opt/logdna-agent-v2
 
 # Install the target image libraries we want to link against.
 # hadolint ignore=DL3008
-RUN apt-get update && apt-get install --no-install-recommends -y dnf
 COPY --from=target /etc/yum.repos.d/ubi.repo /etc/yum.repos.d/ubi.repo
 COPY --from=target /etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release /etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release
-ENV UBI_PACKAGES="systemd-libs systemd-devel glibc glibc-devel gcc libstdc++-devel libstdc++-static kernel-headers"
-RUN dnf install --releasever=8 --forcearch="${TARGET_ARCH}" \
-        --installroot=$SYSROOT_PATH/ --repo=ubi-8-baseos --repo=ubi-8-appstream \
-        --repo=ubi-8-codeready-builder -y $UBI_PACKAGES
 
-RUN printf "/* GNU ld script\n*/\n\
-OUTPUT_FORMAT(elf64-%s)\n\
-GROUP ( /usr/lib64/libgcc_s.so.1  AS_NEEDED ( /usr/lib64/libgcc_s.so.1 ) )" "$(echo ${TARGET_ARCH} | tr '_' '-' )" > $SYSROOT_PATH/usr/lib64/libgcc_s.so
+ENV UBI_PACKAGES="systemd-libs systemd-devel glibc glibc-devel gcc libstdc++-devel libstdc++-static kernel-headers"
 
-# Add the actual agent source files
-COPY . .
+RUN apt-get update && apt-get install --no-install-recommends -y dnf && \
+    dnf install --releasever=8 --forcearch="${TARGET_ARCH}" \
+        --installroot=$SYSROOT_PATH/ --repo=ubi-8-baseos --repo=ubi-8-appstream \
```

Comment:
UBI_VERSION is decoupled from "8..."  here?

Reply:
I have extracted it out as `$UBI_MAJOR_VERSION` I'm not 100% sure package names will remain the same if they ever release a UBI 9, so I've left the script name the same, but it _might_ "just work" if that ever happens.